src/main/java/bio/terra/service/journal/JournalDao.java

```diff
@@ -118,27 +118,24 @@ public void deleteJournalEntryById(@NotNull UUID journalEntryId) {
       isolation = Isolation.SERIALIZABLE,
       readOnly = true)
   public List<JournalEntryModel> retrieveEntriesByIdAndType(
-      UUID resource_key, @NotNull IamResourceType resourceType, long offset, int limit) {
+      UUID resourceKey, @NotNull IamResourceType resourceType, long offset, int limit) {
     String sql =
-        "SELECT "
-            + summaryQueryColumns
-            + "FROM "
-            + TABLE_NAME
-            + " "
-            + "WHERE "
-            + "resource_key = :resource_key "
-            + "AND "
-            + "resource_type = :resourceType "
-            + "ORDER BY entry_timestamp ASC "
-            + "OFFSET :offset LIMIT :limit";
+        """
+            SELECT %s FROM %s
+            WHERE resource_key = :resource_key
+            AND resource_type = :resource_type
+            ORDER BY entry_timestamp DESC
```

Comment:
The only non-formatting change: `DESC` instead of `ASC`.

If we wanted, we could make sort order an input but given the way the upstream endpoint is called I don't think that's necessary right now.